openupgrade_scripts/scripts/hr/16.0.1.1/post-migration.py

```diff
@@ -52,6 +52,9 @@ def create_work_contact(env):
                     ]
                 )
                 nb_matching_partner = len(matching_partner)
+            elif not employee.work_email and not employee.mobile_phone:
```

Comment:
Please put this condition in https://github.com/OCA/OpenUpgrade/pull/5043/files#diff-fbd07eb2ef495367fcf1452043c594e791b1c996976035cb4fd0eae5507a94fcR81